vaadin-archetype-liferay-portlet/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/liferay-display.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<!DOCTYPE display PUBLIC "-//Liferay//DTD Display 6.2.0//EN" "http://www.liferay.com/dtd/liferay-display_6_2_0.dtd">
+
+<display>
+    <category name="${liferayCategoryName}">
+        <portlet id="${portletId}" />
```

Comment:
It seems that `portlet@id` values should match `<portlet-name>` in `liferay-portlet.xml`, otherwise on the UI the portlet is listed in the `Undefined` category.
Perhaps `portletId` property can be completely removed

```suggestion
        <portlet id="${portletName}" />
```